chart/values.yaml

```diff
@@ -1384,7 +1384,7 @@ config:
     remote_logging: '{{- ternary "True" "False" .Values.elasticsearch.enabled }}'
   # Authentication backend used for the experimental API
   api:
-    auth_backends: airflow.api.auth.backend.deny_all
+    auth_backends: airflow.api.auth.backend.session
```

Comment:
This will need to be conditional based on the airflow version.

We might also consider just removing this completely and let the default handle it - the default changed back in 1.10.11, I think it'd be okay as long as we add it to the charts UPDATING.md too.

Reply:
Yeah, lets just remove this in the chart values entirely

Reply:
Removed.